editor-plugin/intellij/build.gradle.kts

```diff
@@ -16,38 +16,26 @@ intellij {
     plugins.set(listOf("JavaScript"))
 }
 
-abstract class GreetingTask : DefaultTask() {
-    @TaskAction
-    fun greet() {
-        println("hello from GreetingTask")
-    }
-}
-
-tasks.register<GreetingTask>("hello")
-
 tasks {
     withType<JavaCompile> {
         sourceCompatibility = "17"
         targetCompatibility = "17"
     }
 
-    named("clean") {
+    clean {
         doFirst {
             exec {
                 commandLine("bash", "-c", "rm -rf *.js")
             }
         }
     }
 
-    named("prepareSandbox") {
+    prepareSandbox {
```

Comment:
Looks even more like we know what we are doing now 😄

Reply:
😂